src/core/validations/error.rs

```diff
@@ -146,6 +146,8 @@ pub enum ValidationError {
     MissingEmbed,
     #[error("timestamp more than 10 mins in the future")]
     TimestampTooFarInFuture,
+    #[error("timestamp more than 1 hour in the past")]
```

Comment:
I'm actually leaning toward making this even tighter, just enough time to account for clock skew. Maybe 10 mins is reasonable.